AUTHORS

```diff
@@ -244,6 +244,7 @@ Levon Saldamli
 Lewis Cowles
 Llandy Riveron Del Risco
 Loic Esteve
+lovetheguitar
```

Comment:
Are you sure you don't want the name here?

Reply:
✅ Fine like it is.